src/Components/ContrastChecker/ContrastChecker.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from "react";
+import "./ContrastChecker.css";
+
+export default function ContrastChecker({ firstColor, secondColor }) {
+  const [contrastResult, setContrastResult] = useState(null);
+
+  useEffect(
+    (contrastResult) => {
+      async function fetchContrast() {
+        try {
+          const response = await fetch(
+            "https://www.aremycolorsaccessible.com/api/are-they",
+            {
+              method: "POST",
+              mode: "cors",
+              body: JSON.stringify({ colors: [firstColor, secondColor] }),
+              headers: {
+                "Content-Type": "application/json",
+              },
+            }
+          );
+
+          if (!response.ok) {
+            throw new Error(
+              `Failed to fetch data! Status Code: ${response.status}`
+            );
+          }
+
+          const contrastData = await response.json();
+          setContrastResult(contrastData.overall);
+
+          console.log("contrastData", contrastData);
+          console.log("contrastResult", contrastResult);
```

Comment:
You can also here remove the consol.log as Jessica said:)

Reply:
Yes, you're right, I'll remove them.